kustomization/components/operator-lifecycle-manager/kustomization.yml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+resources:
+  # renovate: datasource=github-releases
+  - https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.25.0/crds.yaml
+  # renovate: datasource=github-releases
+  - https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.25.0/olm.yaml
```

Comment:
I guess we want to version here and not in the downstream repos?

Reply:
We could publish releases in this repository when we do version bumps here so we can version downstream too.  I think we'll need to play with it a bit to figure things out, but this is a good start.